### ISSUE TITLE
What is the license of this addon?

### ISSUE THREAD

**Issue by ahangarha** [felixfbecker/svg-screenshots#80]
Hi

As per I noticed, there is no license in this repo. This means this addon is a proprietary. This is intended?

**Comment by felixfbecker** [comment]
Yeah I could add one. Curious, how are you trying to use this that is prevented by not having a license, considering it's free in the Chrome store and not really useful as a library?

**Comment by ahangarha** [comment]
One can even use a Source Available software which is not Free/libre and Open Source illegally and unethically. Neither of us would support that (I assume).

If the code be a Free/Libre and Open Source, one can not only use it with freedom, but also can freely contribute to its enhancement.

If a given software is not released under a FLOSS license, the copyright holder can file a case against users and those who modify the code.

I requested license since it seems you wanted to release it as FLOSS. So the missing license should be added to make it legally a Free/Libre and Open Source software. Easy Peasy :)

**Comment by ci7lus** [comment]
Hello. First of all, thanks for such a great extension! It is very useful.
By the way, I would like to publish a rewritten version of this extension as a fork on GitHub (not CWS). This is for personal use. Is it ok?

**Comment by ahangarha** [comment]
> Hello. First of all, thanks for such a great extension! It is very useful. By the way, I would like to publish a bit modified version of this extension as a fork on GitHub (not Chrome Web Store). This is for personal use. Is it ok?

In readme file you can see a badge expressing the license to be MIT. Although the repo lacks LICENSE file, but I think you can do whatever you want based on MIT license. You can make a commercial software out of it as well.

I think I will send a PR for adding a LICENSE file with MIT text to remove such confusion.

**Comment by d262a15b-daf8-4c67-a39d-ec7e03fcc16b** [comment]
Yes, please add the license file to the repository.
That's more explicite than just stating "MIT" in the readme file.
Thank you!
